content/getting-started/white-label-apps.md

```diff
@@ -0,0 +1,96 @@
+---
+description: How to white label apps with Codemagic
+title: White label apps
+weight: 10
+---
+
+White labeling is the practice of building multiple versions of the same application. This can range from one or two up to thousands of different versions. 
+
+This documentation page should be considered as a minimum viable page to help answer most common questions developers have specifically about setting up CI/CD for their white label product in order to understand if they can achieve their desired use case with Codemagic.
+
+The documentation is open-sourced and we encourage you to provide feedback and contributions in the documentation GitHub repository [here](https://github.com/codemagic-ci-cd/codemagic-docs). If you are looking for general documentation about Codemagic please use the search function or browse other parts of the documentation. 
+
+## How to setup Codemagic with ~10 different versions
+
+If you have around 10 different versions of your application that you are trying to set up CI/CD for then consider separating each version into a unique workflow. 
+
+For example, for each version of your app you could create a workflow for it in the codemagic.yaml configuration file. 
+
+```yaml
+workflows:
+  version-one:
+    name: Version one
+    instance_type: mac_mini
+    environment:
+      groups:
+        ...
+      vars:
+        ...
+    scripts:
+      ...
+    artifacts:
+      ...
+    publishing:
+      ...
+  version-two:
+    name: Version two
+    instance_type: mac_mini
+    environment:
+      groups:
+        ...
+      vars:
+        ...
+    scripts:
+      ...
+    artifacts:
+      ...
+    publishing:
+      ...
+```
+
+## How to get started with Codemagic when white labeling ~10 different versions
+
+We would recommend getting familiar with setting up a single version of your app to begin with. Set up your workflow using a codemagic.yaml configuration file and consult the documentation to understand core concepts such as using environment variables, build triggers, script steps, code signing and publishing. Once you have successfully built and published a version of you app to the stores, continue by adding addtional versions to your configuration file. 
```

Comment:
should this perhaps be the first recommendation before advising to set up 10 workflows?